src/CSET/operators/collapse.py

```diff
@@ -109,6 +129,51 @@ def collapse(
                 cube_max = cube.collapsed(coordinate, iris.analysis.MAX)
                 cube_min = cube.collapsed(coordinate, iris.analysis.MIN)
                 collapsed_cubes.append(cube_max - cube_min)
+            elif method == "PROPORTION":
+                match condition:
+                    case "eq":
+                        new_cube = cube.collapsed(
+                            coordinate,
+                            getattr(iris.analysis, method),
+                            function=lambda values: values == threshold,
+                        )
+                    case "ne":
+                        new_cube = cube.collapsed(
+                            coordinate,
+                            getattr(iris.analysis, method),
+                            function=lambda values: values != threshold,
+                        )
+                    case "gt":
+                        new_cube = cube.collapsed(
+                            coordinate,
+                            getattr(iris.analysis, method),
+                            function=lambda values: values > threshold,
+                        )
+                    case "ge":
+                        new_cube = cube.collapsed(
+                            coordinate,
+                            getattr(iris.analysis, method),
+                            function=lambda values: values >= threshold,
+                        )
+                    case "lt":
+                        new_cube = cube.collapsed(
+                            coordinate,
+                            getattr(iris.analysis, method),
+                            function=lambda values: values < threshold,
+                        )
+                    case "le":
+                        new_cube = cube.collapsed(
+                            coordinate,
+                            getattr(iris.analysis, method),
+                            function=lambda values: values <= threshold,
+                        )
+                    case _:
+                        raise ValueError(
+                            """Unexpected value for condition. Expected eq, ne, gt, ge, lt, le. Got {condition}."""
+                        )
+                new_cube.rename(f"probability_of_{cube.name()}_{condition}_{threshold}")
```

Comment:
```suggestion
                name = cube.long_name if cube.long_name else cube.name()
                new_cube.rename(f"probability_of_{name}_{condition}_{threshold}")
```
This will order the names in the same order as we check the colourbars. I.e. "temperature_at_screen_level" before "air_temperature".